boilerplate/fullstack/nuxt/server/utils/convertToRequest.ts

```diff
@@ -0,0 +1,41 @@
+import { H3Event } from "h3";
+import { appInfo } from "~/config/appInfo";
+
+interface ExtendedRequestInit extends RequestInit {
+    duplex?: string;
+}
+
+export async function convertToRequest(event: H3Event): Promise<Request> {
+    const url = new URL(`${appInfo.apiDomain}${event._path}`);
```

Comment:
Can we safely assume that `event._path` will always have a prefix `/`?

Reply:
Yes, the prefix `/` will always be present, as mentioned in the documentation: https://h3.unjs.io/guide/event#eventpath

Reply:
Thanks.